packages/server/src/auth/auth.service.ts

```diff
@@ -75,12 +76,23 @@ export class AuthService {
    * @param email
    * @param password
    * @param resetCode
+   * @returns UpdateStatus
    */
-  async resetPassword(projectId: string, email: string, password: string, resetCode: string): Promise<void> {
-    this.userService.updateUserPassword(projectId, email, password, resetCode);
-    const payload = { message: 'Password updated.' };
+  async resetPassword(projectId: string, email: string, password: string, resetCode: string): Promise<UpdateStatus> {
+    const ok = await this.userService.updateUserPassword(projectId, email, password, resetCode);
```

Comment:
Should probably change this variable from `ok` to something else as it could be something that isn't "ok"

Reply:
I typically use it cause it's readable for boolean or status type checks. But I think it's reasonable to change here.